head.html

```diff
@@ -16,6 +16,6 @@
     )(window, document, "body {opacity: 1 !important}");
 </script>
 <script src="/scripts/lib-franklin.js" type="module"></script>
-<script src="/scripts/scripts.js" type="module"></script>
+<script src="/scripts/load.js" type="module"></script>
```

Comment:
Why the indirection via load.js which in turn calls scripts.js?

Reply:
> Why the indirection via load.js which in turn calls scripts.js?

I had to do this for unit tests. So `feed.js` imports `scripts.js`. As a result, running a UT for `feed.js` implicitly loads `scripts.js` . Now `scripts.js` has `loadPage` getting called as soon as its loaded and this causes the whole front end rendering pipeline  to kick which requires too many mocks et al to run as a UT. And anyway the intent of the test is to just test the `feed` block and not the complete rendering pipeline. 
So I had 2 options:
- Move dependencies from `scripts.js` to some `utils.js` and import that in `feed.js`
- OR, move the `loadPage` to another file so that loading `scripts.js` doesn't end up kicking of the frontend rendering pieline.

The former was too disruptive, requiring lot of refactoring everywhere so I opted for latter. HTH.

Reply:
@sdmcraft You can check [tags.test](https://github.com/hlxsites/sunstar/blob/main/test/blocks/tags/tags.test.js). And i think its not good that for tests only we are changing the default boilerplate behaviour.